OpenSSL.xcconfig

```diff
@@ -1,9 +1,6 @@
 CODE_SIGN_IDENTITY =
-CODE_SIGN_IDENTITY[sdk=macosx*] = Developer ID Application
-CODE_SIGN_IDENTITY[sdk=iphoneos*] = Apple Distribution
-CODE_SIGN_IDENTITY[sdk=iphonesimulator*] = iPhone Distribution
```

Comment:
I wasn't sure if these were needed, or if they'd match exactly what our names are (???), so I just took them out for now. I haven't seen any issues building to simulator or device, but this is definitely something we should monitor once we start trying to create TestFlight builds with these frameworks.